symfony/test/functional/cypress/fixtures/user.json

```diff
@@ -1,8 +1,8 @@
 {
     "admin": {
         "userName": "Admin",
-        "password": "Admin@123",
-        "fullName": "Samantha Jayasinghe"
+        "password": "admin@OHRM123",
+        "fullName": "Chinthika Rathnayake"
```

Comment:
{your-admin-pwd-goes-here}

Reply:
we can above admin user in to `admin_user.json` file

Reply:
Moved admin credentials to the admin.json file.